solana/solana-ibc/programs/solana-ibc/src/tests.rs

```diff
@@ -469,11 +451,12 @@ fn anchor_test_deliver() -> Result<()> {
      */
 
     println!("\nRecving on destination chain");
-    let account_balance_before = sol_rpc_client
-        .get_token_account_balance(&receiver_token_address)
-        .unwrap();
+    let account_balance_before =
+        sol_rpc_client.get_token_account_balance(&receiver_token_address);
+    // .unwrap();
```

Comment:
Why aren’t we unwrapping?

Reply:
Because the account is not yet created, it will be created in the following instruction. But we could also just skip it and use 0 instead.

Reply:
`.unwrap_or_default()` perhaps? Or whatever appropriate `unwrap_or` would be in this case. Though if we expect this to fail (because account is not there yet), let’s just remove this call?

Reply:
```rust
    let account_balance_before = sol_rpc_client
        .get_token_account_balance(&receiver_token_address)
        .map_or(0, |balance| balance.ui_amount.unwrap());
```

should work.  If not assuming zero sounds good to me.

Reply:
LGTM, lets use the above.